docs/man/ntp.toml.5.md

```diff
@@ -179,6 +179,10 @@ ntp-metrics-exporter(8).
     daemon. Levels higher than the given log level are logged as well. If not
     set (the default), then logging will be completely disabled.
 
+`disable-ansi` = `true` | `false` (**false**)
```

Comment:
I'm not sure if this is the best name for this option, I think we should try to not have double negatives (i.e. disable = false). Maybe a better option might be `ansi-colors` with a default value of true.

Reply:
I agree, despite boolean values usually defaulting to false, it's probably better to not have negated options